### PR TITLE
Updated log time stamp format to match RFC-3339

### LINKED ISSUE #223: Standardize Log format timestamp
**Describe the feature request**
Standardize Log format timestamp with [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)
**Is your feature request related to a problem? Please describe**

**Describe alternatives you've considered**

**Additional context**
Example
```sh
{"level":"info","ts":"2022-02-24T19:13:33.570546673Z","logger":"mycontroller","msg":"Pod status updated","pod":{"name":"nginx-1","namespace":"default"},"status":"ready"}
```

### PR DESCRIPTION
**Pull request by rashedkvm**
Signed-off-by: Rashed Kamal <krashed@vmware.com>

Pull request

### Which issue(s) this PR fixes

- Enabled `config-logging`
- Sets `"timeEncoder": "rfc3339nano"`

Fixes #223 

### Describe testing done for PR

Log snippet after config update

```sh
{"level":"info","ts":"2022-03-02T18:19:36.1942387Z","logger":"webhook.ValidationWebhook","caller":"controller/controller.go:538","msg":"Reconcile succeeded","knative.dev/traceid":"a64b6f1e-d543-43bc-abcb-e5b5955b3f11","knative.dev/key":"validation.webhook.bindings.labs.vmware.com","duration":0.0106878}
{"level":"info","ts":"2022-03-02T18:19:36.1963836Z","logger":"webhook.DefaultingWebhook","caller":"controller/controller.go:538","msg":"Reconcile succeeded","knative.dev/traceid":"db874d27-8b62-472f-a7f6-41618e14099f","knative.dev/key":"defaulting.webhook.bindings.labs.vmware.com","duration":0.0122473}
{"level":"info","ts":"2022-03-02T18:19:36.2089394Z","logger":"webhook.ValidationWebhook","caller":"validation/reconcile_config.go:168","msg":"Updating webhook","knative.dev/traceid":"22cc673f-89d0-4f5d-8936-e77ebb1c45aa","knative.dev/key":"validation.webhook.bindings.labs.vmware.com"}
{"level":"info","ts":"2022-03-02T18:19:36.2095734Z","logger":"webhook.DefaultingWebhook","caller":"defaulting/defaulting.go:205","msg":"Updating webhook","knative.dev/traceid":"1c4c548a-a3b2-4e3e-a99f-cc667d919368","knative.dev/key":"defaulting.webhook.bindings.labs.vmware.com"}
{"level":"info","ts":"2022-03-02T18:19:36.2149879Z","logger":"webhook.DefaultingWebhook","caller":"controller/controller.go:538","msg":"Reconcile succeeded","knative.dev/traceid":"1c4c548a-a3b2-4e3e-a99f-cc667d919368","knative.dev/key":"defaulting.webhook.bindings.labs.vmware.com","duration":0.0098519}
{"level":"info","ts":"2022-03-02T18:19:36.2164648Z","logger":"webhook.ValidationWebhook","caller":"controller/controller.go:538","msg":"Reconcile succeeded","knative.dev/traceid":"22cc673f-89d0-4f5d-8936-e77ebb1c45aa","knative.dev/key":"validation.webhook.bindings.labs.vmware.com","duration":0.0119458}
```

### CODE REVIEW

File: config/config-logging.yaml [hunk 1]
```diff
@@ -10,24 +10,7 @@ metadata:
     bindings.labs.vmware.com/release: devel
 
 data:
-  _example: |
```
Comment (scothis) [inline config/config-logging.yaml:13]: it's worth keeping the example as a representation of what the default values are, and what fields are available.

File: config/config-logging.yaml [hunk 2]
```diff
@@ -43,13 +26,8 @@ data:
...
+          "timeEncoder": "rfc3339nano",
           "durationEncoder": "",
           "callerEncoder": ""
         }
       }
-
-    # Log level overrides
-    # Changes are be picked up immediately.
-    loglevel.controller: "info"
-    loglevel.webhook: "info"
```
Comment (scothis) [inline config/config-logging.yaml:55]: these are active config values, why drop them?

File: config/config-logging.yaml [hunk 3]
```diff
@@ -10,24 +10,7 @@ metadata:
...
-    # options and document them in a way that is accessible
-    # to users that `kubectl edit` this config map.
-    #
-    # These sample configuration options may be copied out of
-    # this example block and unindented to be in the data block
-    # to actually change the configuration.
-
-    # Common configuration for all Knative codebase
-    zap-logger-config: |
+  zap-logger-config: |
```
Comment (scothis) [inline config/config-logging.yaml:13]: log output should be sent to stderr, not stdout

File: config/config-logging.yaml (hunk 1 above)
Comment (rashedkvm) [reply config/config-logging.yaml:13]: example added

File: config/config-logging.yaml (hunk 2 above)
Comment (rashedkvm) [reply config/config-logging.yaml:55]: Added those config values

File: config/config-logging.yaml (hunk 3 above)
Comment (rashedkvm) [reply config/config-logging.yaml:13]: Updated

Comment (rashedkvm) [reply config/config-logging.yaml:13]: Are you saying it should be `stderr` for both?

```
"outputPaths": ["stderr"],
"errorOutputPaths": ["stderr"],
```

Comment (rashedkvm) [reply config/config-logging.yaml:13]: I see what you mean, stdout is meant to be program output and stderr is meant to be... "everything else"

File: config/config-logging.yaml [hunk 4]
```diff
@@ -10,6 +10,29 @@ metadata:
...
+          "stacktraceKey": "stacktrace",
+          "lineEnding": "",
+          "levelEncoder": "",
+          "timeEncoder": "rfc3339nano",
+          "durationEncoder": "",
+          "callerEncoder": ""
+        }
+      }
+  loglevel.controller: "info"
+  loglevel.webhook: "info"
```
Comment (scothis) [inline config/config-logging.yaml:35]: best to keep the original content and comments. Make the diff look like you never removed it :)

Comment (rashedkvm) [reply config/config-logging.yaml]: done

**Review by scothis — APPROVED** [review]

**Pull request merged**